compiler/src/dotty/tools/dotc/typer/Inferencing.scala

```diff
@@ -28,11 +28,11 @@ object Inferencing {
    *  but only if the overall result of `isFullyDefined` is `true`.
    *  Variables that are successfully minimized do not count as uninstantiated.
    */
-  def isFullyDefined(tp: Type, force: ForceDegree.Value)(using Context): Boolean = {
+  def isFullyDefined(tp: Type, force: ForceDegree.Value, handleOverflow: Boolean = false)(using Context): Boolean = {
```

Comment:
I just re-read the diff and the "how I fixed it" comment (because I'm looking at handleRecursive and trace) and shouldn't the default here be true and `fullDefinedType` pass true so that it can do it's special handling?  My assumption is that we would want all other calls of `isFullyDefined` to continue to return `false` instead of throwing.

Or are do we want to throw because `isFullyDefined` is call transitively from `IsFullyDefinedAccumulator` and we want those handled by the `handleRecursive` you added?  My point is `handleOverflow` seems to never be non-default, was that for future possible uses?